src/web/views.py

```diff
@@ -267,6 +267,44 @@ def report_hours_detail(request, pk: int):
     )
 
 
+@login_required
+def my_report(request):
+    therapist = request.user
+
+    first_day, last_day = define_month()
+    start_date = request.GET.get("start-date", first_day)
+    end_date = request.GET.get("end-date", last_day)
+
+    start_day = datetime.strptime(start_date, "%Y-%m-%d").date()
+    end_day = datetime.strptime(end_date, "%Y-%m-%d").date()
+
+    massages = (
+        Massage.objects.filter(therapist=therapist)
+        .filter(status="approved")
+        .filter(start__range=(start_day, end_day))
+        .exclude(customer__name="prostovoljec")
+        .order_by("start")
+    )
+
+    amount = 0
+    for massage in massages:
+        amount += massage.service.payout
```

Comment:
You can do this with Django's Sum - https://docs.djangoproject.com/en/4.2/topics/db/aggregation/#following-relationships-backwards
Improve this code in followup PR.

Reply:
[improve code for adding payout amount#145](https://github.com/petmetic/therapy-tracker/issues/145)